src/charts/stackedArea/stackedAreaComponent.js

```diff
@@ -1 +1,24 @@
-console.log('hello stacked area');
+import React, {PureComponent} from 'react';
+import stackedArea from './stackedAreaChart';
+
+
+export default class StackedArea extends PureComponent {
```

Comment:
I would go with regular `Component` just to be safe. `PureComponent` does optimizations based on being able to do shallow comparisons of `props` & `state`. Given that you just have a bucket of `data` and the component is handing off control to Britecharts, it may not actually work

Reply:
OK, thanks!